.github/workflows/ci.yml

```diff
@@ -156,7 +156,7 @@ jobs:
     steps:
     - uses: actions/checkout@v4
     - name: config
-      uses: cross-platform-actions/action@v0.26.0
+      uses: cross-platform-actions/action@fe0167d8082ac584754ef3ffb567fded22642c7d #v0.27.0
```

Comment:
use the tag value?

Reply:
This is the point of this PR. To prevent potential supply chain attacks in GitHub Actions actions revisions from untrusted vendors.